### PR TITLE
coderefinery-mooc: Add basic description of CR MOOC style

### PR DESCRIPTION
**Pull request by rkdarst**
- The main audience here is instructors and those organizing a course,
  so that they will know what to expect.  It also is the first written
  documentation of the plan to us.  The director/producer get training
  by other means.

- This initial PR is designed to provide a shared understanding of the
  goal.  Then, I go and update the other pages (audiece, helper, etc).
- Review
  - Does this make sense to a new reader?
  - What important aspects are missing?
  - Can we give this to an instructor or organizer and say "This is
    what we are doing" ?
  - What important information is missing?

### CODE REVIEW

File: coderefinery-mooc.rst [hunk 1]
```diff
@@ -0,0 +1,186 @@
+CodeRefinery MOOC strategy
+==========================
+
+This page documents the CodeRefinery MOOC strategy.  It is not a real
```
Comment (bast) [inline coderefinery-mooc.rst:4]:
```suggestion
This page documents the CodeRefinery MOOC (massive open online course) strategy.  It is not a real
```

File: coderefinery-mooc.rst [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+Technical setup summary
+-----------------------
+
+We have a public broadcast, with goes out via a livestream.
+Disconnected from this, people are watching the broadcast in a
+*separate* Zoom meeting and doing exercises/breakouts there.  Or,
+people can watch via the livestream alone.  Or there can be different
+meetings.  Or people could watch recorded videos later.
+
+The mental model here is "Watching TV together".  We collective watch
```
Comment (bast) [inline coderefinery-mooc.rst:18]:
```suggestion
The mental model here is "Watching TV together".  We collectively watch
```

File: coderefinery-mooc.rst [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+a show together.  There are periodic intermissions where each watching
+community discusses among themselves and works on the exercises.
+Everyone feels they are a part of something big and that keeps people interested.
+
+We have clear communication channels from learner→instructor
+(HackMD), helpers→instructors (chat), instructors→learners
+(livestream).  Of course instructors can directly to the audience
```
Comment (bast) [inline coderefinery-mooc.rst:25]:
```suggestion
(livestream).  Of course instructors can directly communicate with the audience
```

File: coderefinery-mooc.rst [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+  instructors (and helpers there) can talk without the audience
+  hearing.  This greatly increases professionalism and makes it easier
+  to coordinate.
+* There is the standard text chat (Zulip) to use to communicate with
+  other helpers.
+* Of course, you can go join the student room during breaks, other
+  sessions, and so on.
+
+* Zoom polls won't work.  We can use another service for that instead
+  (e.g. presemo.aalto.fi).
```
Comment (bast) [inline coderefinery-mooc.rst:76]:
```suggestion
  (e.g. presemo.aalto.fi). But also HackMD can be used for polls.
```

**Review by bast — APPROVED** [review]
Maybe I have missed it but later we can also recommend to the instructor to repeat (read back) questions so if we filter/separate audio streams, all watchers can benefit from the questions.

File: coderefinery-mooc.rst (hunk 4 above)
Comment (rkdarst) [reply coderefinery-mooc.rst]: Rephrased differently in follow-up to make HackMD the main emphasis.

**Comment by bast** [comment]
Thanks for this awesome write-up!

**Pull request merged**